types/sqs-consumer/index.d.ts

```diff
@@ -1,29 +1,34 @@
-// Type definitions for sqs-consumer 3.5
+// Type definitions for sqs-consumer 3.7
 // Project: https://github.com/BBC/sqs-consumer
-// Definitions by: Daniel Chao <http://dchao.co/>, Eric Byers <http://www.github.com/EricByers>
+// Definitions by: Daniel Chao <http://dchao.co/>
+//                 Eric Byers <http://github.com/EricByers>
+//                 Ezinwa Okpoechi <http://github.com/BrainMaestro>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
 /// <reference types="node" />
 import { SQS } from "aws-sdk";
 
-export type ConsumerDone = (error?: Error) => void;
+export type ConsumerDone = (error?: SQSError) => void;
 
 export interface Options {
-  queueUrl: string;
-  handleMessage(message: SQS.Message, done: ConsumerDone): any;
-  region?: string;
-  attributeNames?: string[];
-  messageAttributeNames?: string[];
-  batchSize?: number;
-  visibilityTimeout?: number;
-  waitTimeSeconds?: number;
-  authenticationErrorTimeout?: number;
-  sqs?: SQS;
+    queueUrl: string;
+    handleMessage(message: SQS.Message, done: ConsumerDone): any;
+    region?: string;
+    attributeNames?: string[];
+    messageAttributeNames?: string[];
+    batchSize?: number;
+    visibilityTimeout?: number;
+    waitTimeSeconds?: number;
+    authenticationErrorTimeout?: number;
+    sqs?: SQS;
 }
 
-export interface Consumer extends NodeJS.EventEmitter {
-  start(): void;
-  stop(): void;
+declare class SQSError extends Error {}
+
+declare class Consumer extends NodeJS.EventEmitter {
+    start(): void;
+    stop(): void;
+    static create(opts: Options): Consumer;
 }
 
-export function create(opts: Options): Consumer;
+export default Consumer;
```

Comment:
`sqs-consumer` doesn't have a default export.

https://github.com/bbc/sqs-consumer/blob/master/index.js#L203

The correct typing should be `export = Consumer` if we're declaring it as a class.